app/uk/gov/hmrc/economiccrimelevyreturns/controllers/StartController.scala

```diff
@@ -38,6 +38,8 @@ class StartController @Inject() (
     with I18nSupport {
 
   def onPageLoad: Action[AnyContent] = authorise.async { implicit request =>
+    // TODO If the getEclRegistrationDate could return a "maybe date" (Scala Option)
+    //      then upon resulting Scala None we could make this action redirect to "/not-registered"
```

Comment:
You don't need to change anything in this controller - please take a look at `AuthorisedAction` which is where the logic is to determine whether someone is registered or not